core/client/src/main/java/alluxio/client/file/BaseFileSystem.java

```diff
@@ -155,6 +155,7 @@ public void free(AlluxioURI path, FreeOptions options)
     FileSystemMasterClient masterClient = mContext.acquireMasterClient();
     try {
       masterClient.free(path, options);
+      LOG.info("Freed file " + path.getPath());
```

Comment:
In order to keep the same with CreateDirectory or Open command code, are there need options??